client/my-sites/importer/importer-header.jsx

```diff
@@ -92,20 +92,27 @@ export const ImporterHeader = React.createClass( {
 		const canCancel = isEnabled && ! includes( [ appStates.UPLOADING ], importerState );
 		const isScary = includes( [ ...stopStates, ...cancelStates ], importerState );
 
+		// Users were clicking on Stop during artificial delay, which results in backfill failures.
+		// Temporarily introducing this in order to hide Stop button until async job is completed.
```

Comment:
believe it or not :wink:

besides, I didn't say it's permanent, but just that "this is temporary" is usually a lie and rarely defined. I mean, even the Roman empire was temporary